_posts/2024-07-03-csail-network

```diff
@@ -0,0 +1,35 @@
+---
+title: Changes to CSAIL network on July 15, 2024 may affect ESI and MAAS users
+state: active
+# Severity: 0=All systems are operating normally.
+# 1 = There are one or more service interruptions.
+# 2 or more = There is a critical service outage.
+severity: 1
+
+# Prevent template from getting published. Remove this
+# line when copying the template.
+published: false
```

Comment:
I think if you want it published you need to remove this line or add a # before it